bonito/models/configs/dna_r9.4.1@v3.1.toml

```diff
@@ -21,6 +21,8 @@ features = 768
 rnn_type = "lstm"
 activation = "swish"
 blank_score = 2.0
+single_head_layers = [ 2, 3 ]
+single_head_gradient_mult = 1.0
```

Comment:
this shouldn't be needed, but just in case you run into stability, lower this to 0.2